tutorials/xr/setting_up_xr.rst

```diff
@@ -0,0 +1,107 @@
+.. _doc_setting_up_xr:
+
+Setting up XR
+=============
+
+Introduction to the XR system in Godot
+--------------------------------------
+
+Godot provides a moduler XR system that allows different XR platforms to be implemented in such a way that many of the platform specifics are hidden away from the user.
+At the core sits the :ref:`XRServer <class_xrserver>` which acts as a central interface to the XR system that allows users to discover interfaces and interact with the components of the XR system.
+
+Each supported XR platform is implemented as an :ref:`XRInterface <class_xrinterface>`. Supported interfaces register themselves with the :ref:`XRServer <class_xrserver>` and can be queried with the ``find_interface`` method on the :ref:`XRServer <class_xrserver>`. When the desired interface is found it can be initialised by calling ``initialize`` on the interface. 
+
+.. warning::
+	A registered interface means nothing more then that the interface is available, if the interface is not supported by the host system initialization may fail and return ``false``. This can have many reasons and sadly the reasons differ from platform to platform. It can be because the user hasn't installed the required software, or that the user simply hasn't plugged in their headset. You as a developer must thus react properly on an interface failing to initialize. 
```

Comment:
```suggestion
	A registered interface means nothing more than that the interface is available, if the interface is not supported by the host system, initialization may fail and return ``false``. This can have many reasons and sadly the reasons differ from platform to platform. It can be because the user hasn't installed the required software, or that the user simply hasn't plugged in their headset. You as a developer must thus react properly on an interface failing to initialize. 
```